Android/CortexV2Example/README.md

```diff
@@ -0,0 +1,84 @@
+
+## Prerequisites
```

Comment:
you should require to have the client id/secret of app as well
`- The client id/secret of your cortex app (if you don't have, you can register [here](https://www.emotiv.com/my-account/cortex-apps/))`